### PR TITLE
rg-221: Add restrictions without subscription

### PR DESCRIPTION
**Pull request by siefimov**
Added restriction for free subscription users:

- to create more than 1 resume, 
- to download more than 1 resume in PDF format.

image

image

### CODE REVIEW

File: frontend/src/bundles/resume/pages/resume-page.tsx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        }
+    }, [dispatch, userId, isDownloaded]);
+
+    return (
+        <>
+            <div className={styles.resume__wrappers}>
+                <Header>
+                    <div className={styles.resume__actions}>
+                        <RegularButton
+                            // onClick={createResumeAccessLink}
```
Comment (ValentinaTenus) [inline frontend/src/bundles/resume/pages/resume-page.tsx:100]: don't leave comment

File: frontend/src/index.tsx [hunk 2]
```diff
@@ -89,6 +90,18 @@ createRoot(document.querySelector('#root') as HTMLElement).render(
                                             path: AppRoute.TEMPLATES,
                                             element: <TemplatePage />,
                                         },
+                                        {
+                                            path: AppRoute.RESUME,
+                                            element: <ResumePage />,
+                                        },
+                                        {
+                                            path: `${AppRoute.RESUME_EDIT}/:id`,
+                                            element: 'EditResume',
```
Comment (ValentinaTenus) [inline frontend/src/index.tsx:99]: is it correct?

File: shared/src/bundles/resumes/types/resume-views-count-response-dto.type.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+type ResumeViewsCountResponseDto = {
+    resumeId: string;
+    views: number;
+    title: string;
+    image: string;
+    updatedAt: string | undefined;
```
Comment (ValentinaTenus) [inline shared/src/bundles/resumes/types/resume-views-count-response-dto.type.ts:6]:
```suggestion
    updatedAt?: string;
```

File: frontend/src/bundles/resume/pages/resume-page.tsx (hunk 1 above)
Comment (siefimov) [reply frontend/src/bundles/resume/pages/resume-page.tsx:100]: Done.

File: frontend/src/index.tsx (hunk 2 above)
Comment (siefimov) [reply frontend/src/index.tsx:99]: It's just for testing purpose.

Comment (siefimov) [reply frontend/src/index.tsx]: Already changed to real component.

**Comment by vlad-blip** [comment]
[Restrictions without subscription](https://trello.com/c/km3877Sh/221-restrictions-without-subscription)

**Review by vlad-blip — APPROVED** [review]

**Comment by Valentyn13** [comment]
conflicts again :(

**Review by Icerock10 — CHANGES_REQUESTED** [review]
conflicts

**Pull request closed without merging**